cli/dts/lib.deno.ns.d.ts

```diff
@@ -2427,13 +2427,14 @@ declare namespace Deno {
     stdin?: "inherit" | "piped" | "null" | number;
   }
 
-  /** Spawns new subprocess.  RunOptions must contain at a minimum the `opt.cmd`,
+  /** Spawns new subprocess. RunOptions must contain at a minimum the `opt.cmd`,
    * an array of program arguments, the first of which is the binary.
    *
    * ```ts
    * const p = Deno.run({
-   *   cmd: ["echo", "hello"],
+   *   cmd: ["echo", "hello"], // Use ["pwsh", "-c", "echo", "hello"] on Windows.
```

Comment:
This seems to suggest that people should use `Deno.run` with `pwsh` on Windows. To reduce confusion, it would be good to use a command that works on all three operating systems such as `curl`.